Architecture.md

```diff
@@ -12,7 +12,33 @@
 
 #### Web application
 
-***TODO: [CLIENT] fill for web application frontend***
+The website is made using [React](https://react.dev/), we decided to use this framework because it is the most popular, widely used in industry and greatly documented. Furthermore, the whole team was already familiar with React thanks to several past projects.
```

Comment:
Add a source for "most popular, widely used in industry" to prove that point?

```suggestion
The website is made using [React](https://react.dev/). We decided to use this framework because it is the most popular, widely used in industry and greatly documented. Furthermore, the whole team was already familiar with React thanks to several past projects.
```